src/boot/efi/boot.c

```diff
@@ -367,139 +367,118 @@ static UINTN entry_lookup_key(Config *config, UINTN start, CHAR16 key) {
         return -1;
 }
 
+#define PS_FORMAT(prop, fmt, ...) Print(L"%*s: " fmt L"\n", padding, prop, ## __VA_ARGS__)
+#define PS_STRING(prop, str) if (str) { Print(L"%*s: %s\n", padding, prop, str); }
+#define PS_BOOL(prop, value) PS_STRING(prop, yes_no(value))
+
 static VOID print_status(Config *config, CHAR16 *loaded_image_path) {
         UINT64 key;
-        UINTN timeout;
-        BOOLEAN modevar;
-        _cleanup_freepool_ CHAR16 *partstr = NULL, *defaultstr = NULL;
-        UINTN x_max, y_max;
+        BOOLEAN setup_mode = FALSE;
+        UINTN x_max, y_max, padding = 22;
+        _cleanup_freepool_ CHAR16 *device_part_uuid = NULL, *default_efivar = NULL;
+        static const CHAR16 * const random_seed_modes[_RANDOM_SEED_MODE_MAX] = {
+                [RANDOM_SEED_OFF]               = L"off",
+                [RANDOM_SEED_WITH_SYSTEM_TOKEN] = L"with-system-token",
+                [RANDOM_SEED_ALWAYS]            = L"always",
+        };
 
         assert(config);
         assert(loaded_image_path);
 
         clear_screen(COLOR_NORMAL);
         console_query_mode(&x_max, &y_max);
 
-        Print(L"systemd-boot version:   " GIT_VERSION "\n");
-        Print(L"architecture:           " EFI_MACHINE_TYPE_NAME "\n");
-        Print(L"loaded image:           %s\n", loaded_image_path);
-        Print(L"UEFI specification:     %d.%02d\n", ST->Hdr.Revision >> 16, ST->Hdr.Revision & 0xffff);
-        Print(L"firmware vendor:        %s\n", ST->FirmwareVendor);
-        Print(L"firmware version:       %d.%02d\n", ST->FirmwareRevision >> 16, ST->FirmwareRevision & 0xffff);
-        Print(L"console mode:           %d/%ld\n", ST->ConOut->Mode->Mode, ST->ConOut->Mode->MaxMode - 1LL);
-        Print(L"console size:           %d x %d\n", x_max, y_max);
-        Print(L"SecureBoot:             %s\n", yes_no(secure_boot_enabled()));
-
-        if (efivar_get_boolean_u8(EFI_GLOBAL_GUID, L"SetupMode", &modevar) == EFI_SUCCESS)
-                Print(L"SetupMode:              %s\n", modevar ? L"setup" : L"user");
-
-        if (shim_loaded())
-                Print(L"Shim:                   present\n");
-
-        Print(L"OsIndicationsSupported: %d\n", get_os_indications_supported());
-
-        Print(L"\n--- press key ---\n\n");
+        (VOID)efivar_get(LOADER_GUID, L"LoaderDevicePartUUID", &device_part_uuid);
+        (VOID)efivar_get(LOADER_GUID, L"LoaderEntryDefault", &default_efivar);
+        (VOID)efivar_get_boolean_u8(EFI_GLOBAL_GUID, L"SetupMode", &setup_mode);
```

Comment:
doesn't really mattre, but so far we put a space after the (VOID) casts, before the function name